### PR TITLE
BD-3167: Track user sync beta - MERGE 6/18

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-3167**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**June 18. 2024**)
- [ ] No

---

### CODE REVIEW

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+To use this endpoint, you'll need an [API key]({{site.baseurl}}/api/api_key/) with the `users.track.sync` permission.
+
+Customers using the API for server-to-server calls may need to allowlist `rest.iad-01.braze.com` if they're behind a firewall.
+
+## Rate limit
+
+We apply a base speed limit of 500 requests per minute to this endpoint for all customers. Each `/users/sync/track/` request can contain up to one event object, one attribute object, or one purchase object. Each object (event, attribute, and purchase arrays) can update one user each.
+
+{% alert note %}
+Reach out to your customer success manager if you need your limit increased.
```
Comment (bartlam18) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:44]: We should just take this line out, while I definitely want to hear feedback on whether this rate limit is too small, we can't guarantee increasing it.

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+
+## Prerequisites
+
+To use this endpoint, you'll need an [API key]({{site.baseurl}}/api/api_key/) with the `users.track.sync` permission.
+
+Customers using the API for server-to-server calls may need to allowlist `rest.iad-01.braze.com` if they're behind a firewall.
+
+## Rate limit
+
+We apply a base speed limit of 500 requests per minute to this endpoint for all customers. Each `/users/sync/track/` request can contain up to one event object, one attribute object, or one purchase object. Each object (event, attribute, and purchase arrays) can update one user each.
```
Comment (bartlam18) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:41]: Should be /users/track/sync

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+| Parameter | Required | Data Type | Description |
+| --------- | ---------| --------- | ----------- |
+| `attributes` | Optional | One attributes object | See [user attributes object]({{site.baseurl}}/api/objects_filters/user_attributes_object/) |
+| `events` | Optional | One event object | See [events object]({{site.baseurl}}/api/objects_filters/event_object/) |
+| `purchases` | Optional | One purchase object | See [purchases object]({{site.baseurl}}/api/objects_filters/purchase_object/) |
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3  .reset-td-br-4}
+
+## Responses
+
+When using any of the aforementioned API requests, you should receive one of the following two general responses: a successful message or a message with fatal errors.
```
Comment (internetisaiah) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:73]:
```suggestion
When using this endpoint's [request parameters](#request-parameters), you should receive one of the following responses: a successful message or a message with fatal errors.
```

Comment (internetisaiah) [reply _docs/_hidden/private_betas/post_user_track_synchronous.md:73]: ^ the goal being to replace the word `aforementioned`.

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+
+### Successful message
+
+Successful messages will be met with the following response, which includes information about the user profile data that was updated.
```
Comment (internetisaiah) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:77]:
```suggestion
Successful messages will return the following response, which includes information about the user profile data that was updated.
```

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+            ]
+        }
+    ],
+    "message": "success"
+} 
+```
+
+## Frequently asked questions
+
+### In what instances should I use the asynchronous or synchronous endpoint?
```
Comment (internetisaiah) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:272]:
```suggestion
### Should I use the asynchronous or synchronous endpoint?
```

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 6]
```diff
@@ -0,0 +1,292 @@
...
+
+For most profile updates, the `/users/track` endpoint will work best because of its higher rate limit and flexibility to let you batch requests. However, the `/users/track/sync` endpoint is useful if you're experiencing race conditions due to rapid, consecutive requests for the same user.
+
+### Does response time differ compared to the `/users/track` endpoint?
```
Comment (internetisaiah) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:276]:
```suggestion
### Does response time differ from the `/users/track` endpoint?
```

File: _docs/_hidden/private_betas/post_user_track_synchronous.md [hunk 7]
```diff
@@ -0,0 +1,292 @@
...
+
+With a synchronous call, the API waits until the request is completed to return a response. As a result, synchronous requests will take longer on average than asynchronous requests to `/users/track`. For the majority of requests, you can expect a response within seconds.
+
+### Can you send multiple requests to this endpoint at the same time?
```
Comment (internetisaiah) [inline _docs/_hidden/private_betas/post_user_track_synchronous.md:280]:
```suggestion
### Can I send multiple requests to this endpoint at the same time?
```

**Review by internetisaiah — CHANGES_REQUESTED** [review]
Feel free to implement all or some of these changes. The only ones I think that need to be changed are the FAQ ones to:

- reduce heading length
- ensure all questions are using 1st person

**Review by internetisaiah — APPROVED** [review]
LGTM

**Pull request merged**